src/main/java/edu/wpi/grip/core/events/UnexpectedThrowableEvent.java

```diff
@@ -0,0 +1,37 @@
+package edu.wpi.grip.core.events;
+
+/**
+ * Event should be thrown when Unexpected Throwable ends up in an {@link Thread#uncaughtExceptionHandler}.
```

Comment:
Nope, as soon as I'm done with this I'm going to be replacing its invalid usages with something else. In the meantime it stays where it is to handle exceptions we get.